reporter/iface.go

```diff
@@ -51,20 +51,27 @@ type TraceReporter interface {
 }
 
 type ExecutableOpener = func() (process.ReadAtCloser, error)
+type ExecutableMetadataArgs struct {
```

Comment:
Please add a comment to the exported type and exported member variables (parts can be moved here from ExecutableMetadata()).